exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/JoinPruleBase.java

```diff
@@ -246,4 +246,17 @@ public RelNode convertChild(final DrillJoinRel join,  final RelNode rel) throws
 
   }
 
+  // DRILL-6089 make sure no collations are added to HashJoin
+  public static RelTraitSet removeCollation(RelTraitSet traitSet, RelOptRuleCall call)
```

Comment:
Since this is static method, better to add it to PrelUtil class for general purpose use.

Reply:
moved